lib/secret.ts

```diff
@@ -49,3 +53,97 @@ export class SecretFromVariable extends Resource {
     return secret;
   }
 }
+
+export class SecretSet extends Construct {
```

Comment:
I'd vote in favor of naming this a bit more descriptive as its current `forAllLanguages` method seems to sound like this could be called something like `PublishingSecretsSet` or similar.